scripts/install-binary.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+PREFIX=${PREFIX:-/usr/local}
+
+### NOTE: Stripped or replaced by archive.sh from this line
```

Comment:
We shouldn't need these markers

Reply:
Well, it's okay to me, but I thought it's better to mention about `archive.sh` hack. New contributors wouldn't know how this lines are treated.
I thought about separating `install.sh` for release and `install-binary.sh` for debug purpose, but I guess this is easier to maintain.
I can separate them if you prefer that way.😉

Reply:
One script is fine. Can we at least strip those lines from the script that ends up in the zip file then?